roachtest: activerecord failed

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7867050?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7867050?buildTab=artifacts#/activerecord) on master @ [8165e3974c10e88b6ae11c6255872ea16f3a67e3](https://github.com/cockroachdb/cockroach/commits/8165e3974c10e88b6ae11c6255872ea16f3a67e3):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-974-g8165e3974c
Tests run against activerecord 7.0.3
7298 Total Tests Run
7281 tests passed
17 tests failed
26 tests skipped
54 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_change_column_of_parent_table (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_rename_column_of_child_table (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_rename_reference_column_of_child_table (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_reference_column_of_child_table (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #93109 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2 release-blocker]
- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-22214

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7895024?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7895024?buildTab=artifacts#/activerecord) on master @ [24854994805cede37e6845ee2a94e10272b5506b](https://github.com/cockroachdb/cockroach/commits/24854994805cede37e6845ee2a94e10272b5506b):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-1066-g2485499480
Tests run against activerecord 7.0.3
7298 Total Tests Run
7286 tests passed
12 tests failed
26 tests skipped
54 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlTimestampFixtureTest#test_bc_timestamp (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience deprecated-branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/activerecord_blocklist.go
+++ b/pkg/cmd/roachtest/tests/activerecord_blocklist.go
@@ -71,6 +71,7 @@ var activeRecordIgnoreList = blocklist{
 	"LengthValidationTest#test_validates_size_of_association_using_within":                                                                                     "flaky - sometimes complains that a relation does not exist",
 	"PostgresqlInfinityTest#test_where_clause_with_infinite_range_on_a_datetime_column":                                                                        "flaky - sometimes complains that a relation does not exist",
 	"PostgresqlIntervalTest#test_interval_type":                                                                                                                "flaky",
+	"PostgresqlTimestampFixtureTest#test_bc_timestamp":                                                                                                         "flaky - sometimes datetime format does not match",
 	"PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set":                                     "flaky - sometimes complains given Time instead of ActiveSupport::TimeWithZone",
 	"PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set":                       "flaky - sometimes complains given Time instead of ActiveSupport::TimeWithZone",
 	"RelationTest#test_finding_last_with_arel_order":                                                                                                           "flaky - sometimes complains that a relation does not exist",

